### ISSUE TITLE
General questions

### ISSUE THREAD

**Issue by ldoolitt** [ilovb/ProjectOberon2013#7]
First, let me say this project looks great!

Does anyone know if N. Wirth looks at this repository?  Is this done in cooperation with him, or just according to his license?  Would changes here get absorbed by him "upstream", or is there some other "upstream" that is better?

I'm a long-time Verilog programmer, and I started hacking on the Verilog.  Now that I found this repo, it seems that the Verilog files are generated from the provided Lola files.  But I don't see the mechanism for that conversion.  Do I have to do that in an (emulated) Oberon system?  Are there instructions around for how to make that conversion?

I'd like to see this machine run on a larger selection of FPGA boards than that one Digilent board (I have other FPGA boards, but not that one).  My Verilog changes have to do with portability and hardware (FPGA) abstraction, and making it usable in simulation with Icarus Verilog.

I'd ask these questions in private messages to kpmy and/or N. Wirth directly, but don't see a way to do that.

**Comment by congdm** [comment]
As far as I know, the Verilog code of Project Oberon was hand-coded. The Lola-2 code is added after that. You can read more about Lola here: http://www.inf.ethz.ch/personal/wirth/Lola/index.html

**Comment by ldoolitt** [comment]
Yes, I can tell they are different, and the Verilog seems older.  I guess the real question is, which one should be maintained going forward?  Does anyone use the Lola version?  Is Nicklaus still actively developing one or the other?

With some tedium, I can now successfully convert Lola to Verilog using oberon-risc-emu.  Maybe if I get more practice using Oberon it would get easier  Or maybe someone can suggest an Oberon emulator or Lola-Verilog converter more suited for this task.

In the process, I had to fix an incompatibility in hex constants between the posted copies of SPI.Lola and FPDivider.Lola compared to the 20150413 update to the Lola compiler mentioned on N. Wirth's news page.
